app/lib/actions/katello/repository/update_content_urls.rb

```diff
@@ -0,0 +1,15 @@
+module Actions
+  module Katello
+    module Repository
+      class UpdateContentUrls < Actions::EntryAction
+        def plan(content_to_update)
+          concurrence do
+            content_to_update.flat_map(&:repositories).each do |repo|
```

Comment:
I suggested the flat_map approach because I thought we would have to uniq-ify the repos but based on our convo yesterday I don't think its necessary to do this transformation. Can we just do:

```
content_to_update.each do |content|
  content.repositories.each do |repo|
     plan_action ...
  end
end
```